config/eloquent-driver.php

```diff
@@ -18,8 +18,7 @@
 
     'blueprints' => [
         'driver' => 'file',
-        'blueprint_model' => \Statamic\Eloquent\Fields\BlueprintModel::class,
-        'fieldset_model' => \Statamic\Eloquent\Fields\FieldsetModel::class,
+        'model' => \Statamic\Eloquent\Fields\BlueprintModel::class,
```

Comment:
Since these repositories are being split up, they should probably become separate options in the `php please install:eloquent-driver` command.